### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by aneallaryea100**
### Math Magicians: tests

* Set up React Testing Library.
* Follow the instructions in the official [documentation](https://testing-library.com/docs/react-testing-library/intro).
* You don't need to setup Jest (it's included in Create React App).
* Write unit tests for the files operate.js and calculate.js using Jest.
* Create unit tests for all React components:
* Use Jest snapshots to test the components.
* Use React Testing Library to simulate user interaction.
* Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: src/components/Tests/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Renderer from 'react-test-renderer';
+import Calculator from '../Calculator';
+
+it('Renders Calculator component', () => {
+  const tree = Renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (julie-ify) [inline src/components/Tests/Calculator.test.js]: - [ ] You have installed the react testing library, but still need to add tests using it. Kindly write tests simulating user interactions like firing events like clicks and performing calculations on the component. You can link to this tutorial on how to set up tests using the testing library: https://www.robinwieruch.de/react-testing-library/

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @aneallaryea100 @Eternalgratis,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Tests/Calculator.test.js (hunk 1 above)
Comment (aneallaryea100) [reply src/components/Tests/Calculator.test.js]: - [ ] Test added to simulate an event interaction and perform a calculation on component

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**